vega_datasets/dataset_info.json

```diff
@@ -37,6 +37,12 @@
       "Fairfield, Hannah. 'Driving Shifts Into Reverse.' New York Times, May 2, 2010"
     ]
   },
+  "iowa-electricity" : {
+    "description": "The state of Iowa has dramatically increased its production of renewable wind power in recent years. This file contains the annual net generation of electricity in the state by source in thousand megawatthours. The dataset was compiled by the U.S. Energy Information Administration [1]_ and downloaded on May 6, 2018. It is useful for illustrating stacked area charts.",
+    "references": [
+      "https://www.eia.gov/beta/electricity/data/browser/#/topic/0?agg=2,0,1&fuel=vvg&geo=00000g&sec=g&linechart=ELEC.GEN.OTH-IA-99.A~ELEC.GEN.COW-IA-99.A~ELEC.GEN.PEL-IA-99.A~ELEC.GEN.PC-IA-99.A~ELEC.GEN.NG-IA-99.A~~ELEC.GEN.NUC-IA-99.A~ELEC.GEN.HYC-IA-99.A~ELEC.GEN.AOR-IA-99.A~ELEC.GEN.HPS-IA-99.A~&columnchart=ELEC.GEN.ALL-IA-99.A&map=ELEC.GEN.ALL-IA-99.A&freq=A&start=2001&end=2017&ctype=linechart&ltype=pin&tab=overview&maptype=0&rse=0&pin="
+    ]
+  },
```

Comment:
Can you add an explicit note that this data is public domain?

Reply:
It's important that all data bundled with the package is compatible with the package license, and that we document that along the way :smile: